website/siteConfig.js

```diff
@@ -172,6 +172,11 @@ const siteConfig = {
   highlight: {
     // Highlight.js theme to use for syntax highlighting in code blocks.
     theme: 'default',
+    hljs: function(hljs) {
+      hljs.registerLanguage('rbs', function(hljs) {
+        return { disableAutodetect: true }; // Disable RBS highlighting until better options are available, rather than incorrectly highlighting it
+      });
+    },
```

Comment:
Can we set the language to `plaintext` instead of adding this config here?

I worry that if highlight.js ever gains support for RBS, we are going to forget that this config exists here and then be confused why there's no syntax highlighting until someone tracks this line down.

Using `plaintext` at the codeblock means that people won't have to be confused why nothing is ever colored. That's what we do with other codeblocks in Sorbet:

https://github.com/search?q=repo%3Asorbet%2Fsorbet+%2F%60%60%60plaintext%2F&type=code

Reply:
That's fair, will do 👍